python/py-jsonschema/Portfile

```diff
@@ -65,6 +66,17 @@ if {${name} ne ${subport}} {
             depends_lib-append  port:py${python.version}-jsonschema-specifications \
                                 port:py${python.version}-referencing \
                                 port:py${python.version}-rpds-py
+
+            variant format_nongpl description {Build the 'format-nongpl' extra needed for Jupyter 4} {
```

Comment:
the use of `variants` is recommended against unless really necessary. The problem is that you cannot directly depend on a variant, but will need to use the `active_variant` PortGroup which will only check if a port is installed with a variant and, if not, will just throw an error. So if you need to depend on this for another port it should likely not be done with a non-default variant, otherwise the buildbots will never be able to build that port.